crypto/dh/dh_check.c

```diff
@@ -57,13 +57,17 @@ int DH_check_params(const DH *dh, int *ret)
     nid = DH_get_nid((DH *)dh);
     if (nid != NID_undef)
         return 1;
+# ifndef OPENSSL_NO_FIPS186_4_FFC
     /*
      * OR
      * (2b) FFC domain params conform to FIPS-186-4 explicit domain param
      * validity tests.
      */
     return ossl_ffc_params_FIPS186_4_validate(dh->libctx, &dh->params,
```

Comment:
Should the  ossl_ffc_params_FIPS186_4_validate() function be also compiled out?

Reply:
Opened https://github.com/openssl/openssl/issues/25747 as neither have 186_2 ones been compiled out. And there will need to be other code touched as fell re-rearrange for minimum amount of code to build and link.